lib/serve.js

```diff
@@ -294,6 +310,10 @@ module.exports = bootstrap(function createServer(options, config, configFile) {
         : 'No config is used'
     );
 
+    if (!options.prebuild) {
```

Comment:
Looks like it can be joined with next `if` block

Reply:
Fixed